xbmc/utils/CMakeLists.txt

```diff
@@ -177,11 +177,14 @@ if(NOT CORE_SYSTEM_NAME STREQUAL windows)
                       GLUtils.h)
 endif()
 
+if(OPENGLES_FOUND)
+  list(APPEND SOURCES AMLUtils.cpp)
+  list(APPEND HEADERS AMLUtils.h)
+endif()
+
 if(AML_FOUND)
```

Comment:
Don't get it. We only look for AML if ENABLE_AML is set. But if you check with ENABLE_AML here we compile the files even if AML was enabled but not found. Is that what we want?